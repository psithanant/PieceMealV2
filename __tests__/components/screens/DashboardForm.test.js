import 'react-native';
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import TableExample from '../../../src/screens/DashboardForm';

describe('tests TableExample', () => {
  it('renders TableExample component', () => {
  const component = shallow(
    <TableExample
    />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
