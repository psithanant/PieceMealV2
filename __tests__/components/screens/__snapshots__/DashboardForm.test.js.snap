// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests TableExample renders TableExample component 1`] = `
<Styled(Container)>
  <ScrollableTabView
    contentProps={Object {}}
    initialPage={0}
    locked={true}
    onChangeTab={[Function]}
    onScroll={[Function]}
    page={-1}
    prerenderingSiblingsNumber={0}
    scrollWithoutAnimation={false}
    tabBarPosition="top"
  >
    <Styled(Tab)
      activeTextStyle={
        Object {
          "fontSize": 12,
        }
      }
      heading="Profile"
      textStyle={
        Object {
          "fontSize": 12,
        }
      }
    >
      <Connect(Profile) />
    </Styled(Tab)>
    <Styled(Tab)
      activeTextStyle={
        Object {
          "fontSize": 12,
        }
      }
      heading="Statistics"
      textStyle={
        Object {
          "fontSize": 12,
        }
      }
    >
      <Connect(DashboardForm) />
    </Styled(Tab)>
    <Styled(Tab)
      activeTextStyle={
        Object {
          "fontSize": 10,
        }
      }
      heading="Ingredients"
      textStyle={
        Object {
          "fontSize": 11,
        }
      }
    />
    <Styled(Tab)
      activeTextStyle={
        Object {
          "fontSize": 12,
        }
      }
      heading="Recipes"
      textStyle={
        Object {
          "fontSize": 12,
        }
      }
    >
      <Connect(FavoriteRecipes) />
    </Styled(Tab)>
    <Styled(Tab)
      activeTextStyle={
        Object {
          "fontSize": 12,
        }
      }
      heading="Friends"
      textStyle={
        Object {
          "fontSize": 12,
        }
      }
    />
  </ScrollableTabView>
</Styled(Container)>
`;
